### PR TITLE
Remove account specific hardcodings

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock  (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/hms-dbmi-cellenics/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/hms-dbmi-cellenics/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/hms-dbmi-cellenics/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: migrations/sql-migrations/knexfile.js [hunk 1]
```diff
@@ -20,9 +21,14 @@ const fetchConfiguration = async (environment, sandboxId) => {
 
 module.exports = async (env, inputSandboxId) => {
   const environment = env || process.env.NODE_ENV;
-  const sandboxId = inputSandboxId || process.env.SANDBOX_ID || 'default';
+  const sandboxId = inputSandboxId || process.env.SANDBOX_ID;
+  const region = process.env.REGION || 'eu-west-1';
+
+  if (!sandboxId) {
+    throw new Error("Please specify the sandboxId by using the SANDBOX_ID environment variable");
+  }
```
Comment (cosa65) [inline migrations/sql-migrations/knexfile.js:29]: Unrelated, and there's more middle steps than only this, but I think it is still safe to make it so people can't unknowingly run a migration on a `default` cluster, it's better to have them explicitly write `default` if that's the idea

File: infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml [hunk 2]
```diff
@@ -4,8 +4,8 @@ metadata:
   namespace: default
   name: 503-for-pending-subscriptions
   annotations:
-    alb.ingress.kubernetes.io/certificate-arn: >-
-      arn:aws:acm:eu-west-1:242905224710:certificate/de2efba9-8964-49d2-8525-242e7e43ca4c
+    alb.ingress.kubernetes.io/certificate-arn: >- 
+      {{ .Values.myAccount.acmCertificate }}
```
Comment (cosa65) [inline infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml:8]: I am not really sure about this change, I copied how this was set in `charts/nodejs/templates/ingress.yaml` but would like to go over it with someone to make sure it is ok

File: .github/workflows/deploy-infra.yaml [hunk 3]
```diff
@@ -254,7 +254,7 @@ jobs:
           yq w -d0 infra/account-config.yaml myAccount.domainName --style=double "$DOMAIN_NAME" \
           | yq w -d0 - myAccount.region --style=double "$AWS_REGION" \
           | yq w -d0 - myAccount.accountId --style=double "$AWS_ACCOUNT_ID" \
-          | yq w -d0 - myAccount.acmCertificate --style=double "arn:aws:acm:eu-west-1:242905224710:certificate/de2efba9-8964-49d2-8525-242e7e43ca4c" \
```
Comment (ivababukova) [inline .github/workflows/deploy-infra.yaml:257]: there is ACM_CERTIFICATE_ARN in secrets. You should delete the `"arn:aws:acm:eu-west-1:242905224710:certificate/de2efba9-8964-49d2-8525-242e7e43ca4c"` bit entirely and just read the arn from that secret

File: infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml (hunk 2 above)
Comment (ivababukova) [reply infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml:8]: yup, this is correct. This should have been fixed before, there was a ticket for it. I am surprised that we skipped it. Thank you for finding it!

File: migrations/sql-migrations/knexfile.js (hunk 1 above)
Comment (ivababukova) [reply migrations/sql-migrations/knexfile.js:29]: I agree

**Review by ivababukova — APPROVED** [review]

**Pull request merged**